### PR TITLE
add-js-codebytes

### LINKED ISSUE #2279: [Edit] JavaScript Add Codebytes
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Please add Codebytes to the listed entries.
https://www.codecademy.com/resources/docs/javascript/dates/now
https://www.codecademy.com/resources/docs/javascript/dates/toLocaleDateString

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) when working on the PR for this issue.

Bear in mind the following two principles:
1) There should only be one Codebyte per entry listed as `## Codebyte Example` at the end of the entry.
2) The codebytes should be new examples, not copies of the prior examples.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by nikolaspoczekaj**
### Description

Thanks for assigning. This closes #2279 . 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@nikolaspoczekaj Looks good! Sending to second review.

File: content/javascript/concepts/dates/terms/now/now.md [hunk 1]
```diff
@@ -54,3 +54,16 @@ console.log(monthsSince1970);
...
 // Note: your output may vary
 ```
+
+## Codebyte Example
+
+The following executable example returns the current timestamp in milliseconds since January 1, 1970 00:00:00 UTC:
+
+> **Note**: The code will print different values each time it is executed.
+
+```codebyte/javascript
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/now/now.md:64]: In the new example please remove the extra empty lines at the top and bottom of the code block

File: content/javascript/concepts/dates/terms/toLocaleDateString/toLocaleDateString.md [hunk 2]
```diff
@@ -87,3 +87,23 @@ The output will be:
 ```shell
 Monday April 11, 2022
 ```
+
+## Codebyte Example
+
+In the following example the variables `region` and `options` can be modified, to print the Date in a custom format:
+
+```codebyte/javascript
+
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/toLocaleDateString/toLocaleDateString.md:96]: As noted above, please remove the additional empty lines at the top and bottom of the code block

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@nikolaspoczekaj I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@nikolaspoczekaj Looks good. I just made a couple of minor edits to the now.md entry.

**Comment by SSwiniarski** [comment]
@nikolaspoczekaj Congrats on the new edits!

Live links here:
- https://www.codecademy.com/resources/docs/javascript/dates/now
- https://www.codecademy.com/resources/docs/javascript/dates/toLocaleDateString

**Pull request merged**